Source/AwsCommonRuntimeKit/http/HTTPStream.swift

```diff
@@ -3,7 +3,7 @@
 import AwsCHttp
 import Foundation
 
-/// An base class that represents a single Http Request/Response for both HTTP/1.1 and HTTP/2.
+/// An base abstract class that represents a single Http Request/Response for both HTTP/1.1 and HTTP/2.
 /// Can be used to update the Window size, and get status code.
 public class HTTPStream {
```

Comment:
I thought about turning this class into a protocol to avoid the abstract class workaround with `fatalAssert`, but that would require making `rawValue` and `callbackData` variables public.